.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -509,13 +533,66 @@ jobs:
           output-folder: "${{ env.OUTPUT_FOLDER }}-${{ needs.rng.outputs.value }}"
 
       - name: Upload the signed attestations
-        id: upload
+        id: upload-signed
         if: ${{ github.event_name != 'pull_request' }}
-        uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/secure-upload-artifact
         with:
           name: "${{ env.OUTPUT_FOLDER }}-${{ needs.rng.outputs.value }}"
```

Comment:
do we verify these thru secure download, or that's part of https://github.com/slsa-framework/slsa-github-generator/issues/1655? if os, maybe also add a comment here with the issue

Reply:
there is a comment with the issue where the verification should go (line 564)